submissions/annStoyano/html-css-popup/index.html

```diff
@@ -0,0 +1,155 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="css/style.min.css">
+</head>
+
+<body>
+    <header class="header">
+        <nav class="navigation">
+            <a href="#" class="mails link">Mails</a>
+            <a href="#" class="mails link">Images</a>
+            <section class="popup-wrapper">
+                <input type="checkbox" id='popup-check' class='popup_btn'>
+                <label class="popup_btn-icon" for='popup-check'></label>
```

Comment:
Navigation is best used in this format, the sample from [MDN](https://developer.mozilla.org/uk/docs/Web/HTML/%D0%95%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82/nav)
```
<nav>
  <ul>
    <li><a href="#">Home</a></li>
    <li><a href="#">About</a></li>
    <li><a href="#">Contact</a></li>
  </ul>
</nav>
```